### PR TITLE
Warn user if trying to upload ZIP instead of a folder from backup

### LINKED ISSUE #293: Tooltip when restoring a backup needs to be more clear or should accept zip files
Screen Shot 2020-08-20 at 00 17 05

This tooltip needs to either mention you need to extract the zipfile to restore the wallet to avoid some confusion

### PR DESCRIPTION
**Pull request by ben-kaufman**
Allowing to upload a zip would have been better, but the way it is currently built it is a bit problematic, so for now at least just tell the user to unzip and select the extracted folder.
Fix #293

### CODE REVIEW

**Comment by stepansnigirev** [comment]
Actually I can't select .zip file from the backup, so I never see the warning.
Maybe makes sense to add a note to the info tooltip as well?

**Comment by ben-kaufman** [comment]
Ok, added a note on that there.

File: src/cryptoadvance/specter/templates/settings/general_settings.jinja [hunk 1]
```diff
@@ -44,6 +44,7 @@
 					<span class="info">
 						<span class="info__title">Restoring Specter from backup<br><br></span>
 						Here you can restore your wallets and devices from an existing Secter backup.<br>
+						Please make sure to unzip the backup file first, then to select the exctarcted folder for upload.<br>
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/templates/settings/general_settings.jinja:47]: exctarcted =)

Comment (ben-kaufman) [reply src/cryptoadvance/specter/templates/settings/general_settings.jinja]: 🤦‍♂️  right, fixed

**Pull request merged**